configure

```diff
@@ -323,6 +323,10 @@ fi
 
 inf "Using compiler $cc."
 
+# Configure compiler to use in further tests.
+
+export cc="$cc"
```

Comment:
You can/should write only

    export cc

More portable (even though we don't care about systems with shells that don't handle this) and consistent.